src/background/service-worker-init.ts

```diff
@@ -46,7 +47,15 @@ import { cleanKeysFromStorage } from './user-stored-data-cleaner';
 async function initialize(): Promise<void> {
     const userAgentParser = new UAParser(globalThis.navigator.userAgent);
     const browserAdapterFactory = new BrowserAdapterFactory(userAgentParser);
-    const browserAdapter = browserAdapterFactory.makeFromUserAgent();
+    const logger = createDefaultLogger();
+    const promiseFactory = createDefaultPromiseFactory();
+    const FIRE_AND_FORGET_EVENT_DELAY_MS = 2 * 60 * 1000; // 2 minutes
```

Comment:
Without the comment explaining its purpose near it, this feels like a magic number and it's hard to understand why 2 minutes makes sense for the value. I feel like it's easier for a reason to understand whether 2 minutes sounds reasonable if it's explained in the context of the rest of the browser event manager code, so my preference would be to leave the constant there and to use a boolean `isServiceWorker` for the ctor parameter.